zerver/tests/test_messages.py

```diff
@@ -163,10 +163,15 @@ def setUp(self):
         dep.realms = [get_realm("zulip")]
         dep.save()
 
+    def create_bot(self, email):
+        # type: (Text) -> UserProfile
+        self.register(email, 'test')
+        return get_system_bot(email)
```

Comment:
I think we shouldn't use `get_system_bot` here, since semantically, these are user-created bots, not system bots.